aspnetcore/blazor/hosting-models.md

```diff
@@ -18,9 +18,34 @@ NOTE: Daggered lines under the table (&dagger;, &Dagger;) use a double-space at
 
 -->
 
-This article explains the different Blazor hosting models and how to choose which one to use.
+:::moniker range=">= aspnetcore-8.0"
 
-:::moniker range=">= aspnetcore-6.0"
+This article explains Blazor hosting models, which can be applied in different parts of a Blazor app at either compile time or runtime.
+
+:::moniker-end
+
+:::moniker range="< aspnetcore-8.0"
+
+This article explains Blazor hosting models and how to choose which one to use.
+
+:::moniker-end
+
+:::moniker range=">= aspnetcore-8.0"
+
+Blazor is a web framework for building web UI components ([Razor components](xref:blazor/components/index)) that can be hosted in different ways:
+
+* Server-side in ASP.NET Core (*Blazor Server*).
+* Client-side in the browser on a [WebAssembly](https://webassembly.org/)-based .NET runtime (*Blazor WebAssembly*, *Blazor WASM*).
+* Determined at runtime, starting with server-side (*Blazor Server*) and switching over to client-side (*Blazor WebAssembly*) after the components, their dependencies, and the .NET runtime are downloaded to the browser.
```

Comment:
Hmm...I don't think I would include this one in the hosting model list. This is really about changing how a component is rendered at runtime, which I assume is covered in detail in the render modes doc.